src/components/users/Accounts.jsx

```diff
@@ -55,6 +58,33 @@ export const accountsToDbusUsers = (accounts) => {
     }];
 };
 
+const reservedNames = [
```

Comment:
This code now is duplicated pyanaconda/core/users.py

I will create an issue to expose it via an API and remove the duplication.

Reply:
https://issues.redhat.com/browse/INSTALLER-3839